configuration/parse.go

```diff
@@ -99,6 +108,9 @@ func Parse(args []string) (Configuration, error) {
 	flag.BoolVar(&config.Quiet, "quiet", false, "indicate output should contain only packages with vulnerabilities")
 	flag.BoolVar(&config.Version, "version", false, "prints current nancy version")
 	flag.BoolVar(&config.CleanCache, "clean-cache", false, "Deletes local cache directory")
+	flag.BoolVar(&config.Info, "v", false, "Set log level to Info")
+	flag.BoolVar(&config.Debug, "vv", false, "Set log level to Debug")
+	flag.BoolVar(&config.Trace, "vvv", false, "Set log level to Trace")
```

Comment:
We have a bunch of testing around parsing these and the effects of them probably should add some around to make sure the v's are doing what we expect.

Reply:
NOOOOOOOOO (ok I'll do it)

Reply:
In 3dce67a and e3a3fb1